pkg/scheduler/api/pod_info.go

```diff
@@ -149,6 +151,11 @@ func GetPodResourceWithoutInitContainers(pod *v1.Pod) *Resource {
 		result.Add(NewResource(container.Resources.Requests))
 	}
 
+	// if PodOverhead feature is supported, add overhead for running a pod
+	if pod.Spec.Overhead != nil && utilfeature.DefaultFeatureGate.Enabled(features.PodOverhead) {
```

Comment:
how to enable/disable this feature with volcano?

Reply:
It is enabled on Kubernetes. It needs to make sure the `PodOverhead` feature gate is enable and it is enabled by default on kubernetes version above 1.18. `overhead` can be defined on `RuntimeClass`.
Feature gate can be enabled by adding `--feature-gates` flag.